hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -492,6 +492,17 @@
 	String USE_NEW_ID_GENERATOR_MAPPINGS = "hibernate.id.new_generator_mappings";
 
 	/**
+	 * Setting which indicates whether or not the new type conflict resolving should be used.
+	 * <p/>
+	 * Default is {@code true}.  Existing applications may want to disable this (set it {@code false}) for
+	 * upgrade compatibility.
+	 *
+	 * @see MetadataBuilder#enableNewTypeConflictResolverSupport
+	 * @since 5.2.9
+	 */
+	String USE_NEW_TYPE_CONFLICT_RESOLVER = "hibernate.id.new_type_conflict_resolver";
+
+	/**
```

Comment:
existing application should not be affected by a micro release so the default should be false.